src/sql/base/browser/ui/table/media/table.css

```diff
@@ -5,6 +5,7 @@
 
 .monaco-table * {
 	box-sizing: border-box;
+	white-space: pre;
 }
```

Comment:
@anthonydresser please let me know if this is right place to add this

Reply:
I would just double check it doesn't obviously break any of the other places the tables class is used.

Reply:
Checked jobs/profilers/editdata/insights/DacFx etc. usage. Do let me know if there is something specific you think I should check..